# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/encoder/MediaEncoder.cpp
# Commit: 398d14785168
# Full Hash: 398d147851684faa0f397a2e206e83f6b0b98f02
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-10-02 16:35:54
# Regressor Bug: 1493613
# File Overlap Count: 1
# Description:
#   Bug 1454998 - Remove track blocking from MediaInputPort. r=karlt
#   
#   This functionality is not used since we moved to only having a single track per
#   MediaStream (bug 1493613).
#   
# ==============================================================================

diff -r a4d88d5b86d2 -r 398d14785168 dom/media/encoder/MediaEncoder.cpp
--- a/dom/media/encoder/MediaEncoder.cpp	Wed Oct 02 10:22:48 2019 +0000
+++ b/dom/media/encoder/MediaEncoder.cpp	Wed Oct 02 10:22:49 2019 +0000
@@ -522,8 +522,8 @@
     mPipeStream = AudioNodeStream::Create(ctx, engine, flags, ctx->Graph());
     AudioNodeStream* ns = aNode->GetStream();
     if (ns) {
-      mInputPort = mPipeStream->AllocateInputPort(aNode->GetStream(), TRACK_ANY,
-                                                  TRACK_ANY, 0, aOutput);
+      mInputPort =
+          mPipeStream->AllocateInputPort(aNode->GetStream(), 0, aOutput);
     }
   }
 